manifest-parser.js

```diff
@@ -224,12 +225,14 @@ var ManifestParser = (function() {
   function _parse(string) {
     // TODO: temporary while ManifestParser is a collection of static methods.
     _logs = [];
+    _tips = [];
     _success = true;
 
     try {
       _json_input = JSON.parse(string);
     } catch (e) {
       _logs.push("File isn't valid JSON: " + e);
+      _tips.push('Your JSON failed to parse, these are the main reasons why JSON parsing usually fails:\n - Double quotes should be used around property names and for strings. Single quotes are not valid.\n - JSON specification disallow trailing comma after the last property even if some implementations allow it.');
```

Comment:
can we break this string into multiple lines < 80chars ... otherwise this will fail on the jscs style lint